packages/workflow/src/records/fhir.ts

```diff
@@ -798,6 +798,44 @@ export function createUnassignedTask(
   return unassignedTask
 }
 
+export function createVerifyRecordTask(
+  previousTask: SavedTask,
+  headers: IAuthHeader
+): SavedTask {
+  return {
+    resourceType: 'Task',
+    status: 'accepted',
+    intent: 'proposal',
+    code: previousTask.code,
+    focus: previousTask.focus,
+    id: previousTask.id,
+    identifier: previousTask.identifier,
+    extension: [
+      ...previousTask.extension.filter((extension) =>
+        [
+          'http://opencrvs.org/specs/extension/contact-person-phone-number',
+          'http://opencrvs.org/specs/extension/informants-signature',
+          'http://opencrvs.org/specs/extension/contact-person-email',
+          'http://opencrvs.org/specs/extension/bride-signature',
+          'http://opencrvs.org/specs/extension/groom-signature',
+          'http://opencrvs.org/specs/extension/witness-one-signature',
+          'http://opencrvs.org/specs/extension/witness-two-signature'
```

Comment:
Why is it exactly these URLs that are filtered in?

Reply:
We sort of create new task from scratch for every action e.g. update, validate, register etc. If we don't keep them by filtering like this, we lose these data after each action.

Reply:
What would happen if it wasn't filtered and only just `extension: previousTask.extension`? Are there some extra extensions here that would get duplicated? 🤔 

If a new developer comes and sees this array of strings, would it be easy to understand why it's exactly these values that are filtered in? If possible, I'd move these to a variable that could ideally be `as SomeType[]` or `satisfies SomeType[]` to make it a bit clearer

Reply:
Yeah, we probably should. I have a PR open for refactoring the creation of tasks. Maybe, this can be done on that PR. 

And if we just set it to `extension: previousTask.extension`, without filtering, the latest action of the record wouldn't be displayed correctly. So, we filter out the extensions like, `downloaded`, `marked-as-duplicate` and only keep the latest ones and also the important information like the signatures. Maybe, this would help to clear things up.
https://github.com/opencrvs/opencrvs-core/blob/feature/state-transitions/packages/commons/src/fhir/task.ts#L247

Reply:
Nice, thanks! Makes a lot of sense